### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by roman-zherebetskyi**

### CODE REVIEW

File: src/calculateRentalCost.js [hunk 1]
```diff
@@ -28,7 +28,17 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const result = days * 40;
+
+  if (days > 2 && days < 7) {
+    return result - 20;
```
Comment (Andr1yK) [inline src/calculateRentalCost.js:34]: what is 20, 50... or 40? I don't know🥲

File: src/calculateRentalCost.js [hunk 2]
```diff
@@ -28,7 +28,17 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const result = days * 40;
+
+  if (days > 2 && days < 7) {
```
Comment (Andr1yK) [inline src/calculateRentalCost.js:33]: u can check >=7 days and avoid &&

**Review by Andr1yK — CHANGES_REQUESTED** [review]

File: src/calculateRentalCost.js (hunk 1 above)
Comment (roman-zherebetskyi) [reply src/calculateRentalCost.js]: okay I'll create a variables

File: src/calculateRentalCost.js [hunk 3]
```diff
@@ -28,7 +28,25 @@
...
+
+  if (days < 7) {
+    return result - lowDiscount;
+  }
+
+  if (days >= 7) {
+    return result - bigDiscount;
+  }
+
+  return result;
```
Comment (Andr1yK) [inline src/calculateRentalCost.js:49]: u really need 4 returns? Mb, we can decrease their count?)

**Review by Andr1yK — CHANGES_REQUESTED** [review]

File: src/calculateRentalCost.js (hunk 3 above)
Comment (roman-zherebetskyi) [reply src/calculateRentalCost.js]: Andreeeeeeeew!!!!

Comment (Andr1yK) [reply src/calculateRentalCost.js]: oops😂

**Review by Andr1yK — APPROVED** [review]
GJ

**Pull request closed without merging**